ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane-savage.ts

```diff
@@ -346,6 +473,209 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'ASSS Fresh Puff 2 Bait', // 4 puffs on cardinals or intercardinals with tethers
+      type: 'Tether',
+      netRegex: { source: 'Silken Puff' },
+      condition: (data, matches) => matches.target === data.me && data.puffCounter === 2,
+      durationSeconds: 6,
+      alertText: (data, matches, output) => {
+        const dirCards = ['N', 'E', 'S', 'W'];
+        let silkieStatus = '';
+        switch (data.suds) {
+          case 'CE1': // Middle Safe
+            silkieStatus = 'bossWind';
+            break;
+          case 'CE2': // Intercards Safe
+            silkieStatus = 'bossIce';
+            // never CE3 (lightning) for this mechanic
+        }
+        if (silkieStatus === '')
+          return output.default!();
+
+        const tetheredPuff = data.silkenPuffs[matches.sourceId];
+        if (tetheredPuff === undefined)
+          return;
+
+        // See Silken Puff Suds Gain trigger for list of Silken Puff effectIds
+        // Puff must be either CEA (Ice / blue) or CEB (Lightning / yellow) in this mechanic
+        const puffEffect = tetheredPuff.effect === 'CEA' ? 'Blue' : 'Yellow';
+        const puffDir = tetheredPuff.location;
+        if (puffDir === undefined)
+          return output.default!();
+
+        const puffLocs = dirCards.includes(puffDir) ? 'Cardinal' : 'Intercard';
+        const baitOutput: string = silkieStatus + puffEffect + puffLocs + 'Puff';
+        const safeOutput: string = silkieStatus + 'Puffs' + puffLocs + 'SafeLater';
+
+        // set the output for the subsequent safe call here and pass the output to the followup trigger
+        // this keeps all of the interrelated output strings in this trigger for ease of customization
+        data.freshPuff2SafeAlert = output[safeOutput]!();
+        return output.bait!({
+          boss: output[silkieStatus]!(),
+          dir: puffDir,
```

Comment:
missing output[] here?:
dir: output[puffDir]!(),

Reply:
Yeah, good catch.  I forgot to localize that.